### DISCUSSION TITLE
It's possible to update the new version, on the admin dashboard.

### DISCUSSION THREAD

**Discussion by MehediHasannOvi** [pocketbase/pocketbase#2580] [Q&A]
It's possible to update the new version, on the admin dashboard. I'm currently using digital ocean Install Pocketbase I can't update to the latest version

**Comment by ganigeorgiev** [comment]
If you are using PocketBase v0.16+, you can SSH to your server and run `/path/to/pocketbase update` (there is no UI for this at the moment). This will download the latest binary and replace your current executable.

If you are using an older version you'll have to replace the executable manually (eg. using `scp`, `rsync`, etc.), Here is an example with `rsync`:

```sh
rsync -avz -e ssh /local/path/to/pocketbase root@YOUR_SERVER_IP:/remote/path/to/pocketbase
```

**Comment by MehediHasannOvi** [comment]
how can I update on Digitalocean, Digitalocean By default Version 0.14.4, I try to update on default Console ./pocketbase update, but it's not updated,

**Reply by tameraktasTU** [reply]
You can only use ./pocketbase update with version 0.16.0 and up.
